lib/fs.ts

```diff
@@ -25,7 +25,9 @@ import * as fs from 'fs';
 import * as pathModule from 'path';
 import {ReadStream, WriteStream} from './streams';
 
-const ROOT_PATH = pathModule.resolve(__dirname, '..');
+// account for pwd/dist/lib
+const DIST = `${pathModule.sep}dist${pathModule.sep}`;
```

Comment:
Is this `sep` stuff necessary? Windows accepts `/` and I think `/` is used elsewhere in the code anyway

Reply:
In my tests with Wob, it was an issue. Not sure why it's not necessary other places - I think Node resolves / to path.sep?

Reply:
Ok that's fine, we don't have to dwell on it given that we need to ship this but put a comment explaining the situation I guess